docs/modules/user-guide/partials/proc_devfile-registry.adoc

```diff
@@ -0,0 +1,45 @@
+[id="proc_devfile-registry_{context}"]
+= Devfile registry
+
+[role="_abstract"]
+A devfile registry is a service that stores and provides devfile stacks to Kubernetes developer tools like odo, Eclipse Che and the OpenShift Developer Console. Therefore, you can access devfiles through the devfile registry.  Devfile registries are based on the Oracle Cloud Infrastructure (OCI) Specification, and devfile stacks are stored as OCI-artifacts in the registry.
+
+Each devfile stack corresponds to a specific runtime or framework, such as,  NodeJS, Quarkus, and Wildfly. A devfile stack also contains resources, such as, a devfile.yaml, logo, and outerloop resources. These devfile stacks give developers a template so they can get started developing cloud-native applications.
```

Comment:
Is "outerloop" Red Hat jargon or is it a devfile term that is well-understood users?

Reply:
Good point. @johnmcollier, I'm thinking users will be unfamiliar with `logo` and `outerloop resources`; therefore, maybe we should remove them. What do you think?